dnf-behave-tests/dnf/repo-sync.feature

```diff
@@ -33,17 +42,16 @@ Scenario: There is global skip_if_unavailable option
         | baseurl         | /non/existent/repo |
    When I execute dnf with args "makecache"
    Then the exit code is 0
-    And stdout matches line by line
+    And stdout is
     """
-    testrepo
-    Metadata cache created\.
+    Metadata cache created.
     """
-    And stderr is
+    And stderr matches line by line
     """
-    Errors during downloading metadata for repository 'testrepo':
-      - Curl error (37): Couldn't read a file:// file for file:///non/existent/repo/repodata/repomd.xml [Couldn't open file /non/existent/repo/repodata/repomd.xml]
-    Error: Failed to download metadata for repo 'testrepo': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
-    Ignoring repositories: testrepo
+    <REPOSYNC>
+    >>> Curl error \(37\): Could.*t read a file:// file for file:///non/existent/repo/repodata/repomd.xml \[Couldn't open file /non/existent/repo/repodata/repomd.xml\] - file:///non/existent/repo/repodata/repomd.xml - repomd.xml
```

Comment:
You could also replace the cryptic `Could.*t` regex with `(Couldn't|Could not)` to make it clearer.

Reply:
That is better, thank you!
Updated.